spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -178,6 +178,96 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testFindRelatedStreams() throws Exception {
```

Comment:
mvc tests tend to be less readable IMO. Also, we should test the actual logic. The fact that business logic is constantly added to the *controller* is wrong in the first place if you ask me..